src/main/java/seedu/taassist/logic/commands/ListCommand.java

```diff
@@ -8,7 +8,7 @@
 import seedu.taassist.model.student.predicate.IsPartOfClassPredicate;
 
 /**
- * Lists all students in TA-Assist or in the focused class to the user.
+ * Lists all students in TA-Assist or in the focused class.
```

Comment:
```suggestion
 * Lists all students in TA-Assist or in the currently focused class.
```

Minor, but for consistency reasons.